src/main/java/uk/gov/hmcts/reform/hmc/repository/DefaultFutureHearingRepository.java

```diff
@@ -1,29 +1,49 @@
 package uk.gov.hmcts.reform.hmc.repository;
 
+import com.fasterxml.jackson.databind.JsonNode;
 import org.springframework.stereotype.Repository;
 import uk.gov.hmcts.reform.hmc.ApplicationParams;
+import uk.gov.hmcts.reform.hmc.client.futurehearing.ActiveDirectoryApiClient;
 import uk.gov.hmcts.reform.hmc.client.futurehearing.AuthenticationRequest;
 import uk.gov.hmcts.reform.hmc.client.futurehearing.AuthenticationResponse;
-import uk.gov.hmcts.reform.hmc.client.futurehearing.FutureHearingApiClient;
+import uk.gov.hmcts.reform.hmc.client.futurehearing.HearingManagementInterfaceApiClient;
+import uk.gov.hmcts.reform.hmc.client.futurehearing.HearingManagementInterfaceResponse;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.springframework.http.MediaType.APPLICATION_JSON_VALUE;
 
 @Repository("defaultFutureHearingRepository")
 public class DefaultFutureHearingRepository implements FutureHearingRepository {
 
-    private final FutureHearingApiClient futureHearingApiClient;
+    private final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss'Z'");
+    private final HearingManagementInterfaceApiClient hmiClient;
+    private final ActiveDirectoryApiClient activeDirectoryApiClient;
     private final ApplicationParams applicationParams;
 
-    public DefaultFutureHearingRepository(FutureHearingApiClient futureHearingApiClient,
-                                          ApplicationParams applicationParams) {
-        this.futureHearingApiClient = futureHearingApiClient;
+    public DefaultFutureHearingRepository(ActiveDirectoryApiClient activeDirectoryApiClient,
+                                          ApplicationParams applicationParams,
+                                          HearingManagementInterfaceApiClient hmiClient) {
+        this.activeDirectoryApiClient = activeDirectoryApiClient;
         this.applicationParams = applicationParams;
+        this.hmiClient = hmiClient;
     }
 
     public AuthenticationResponse retrieveAuthToken() {
-        return futureHearingApiClient.authenticate(
+        return activeDirectoryApiClient.authenticate(
             new AuthenticationRequest(
                 applicationParams.getGrantType(),
                 applicationParams.getClientId(), applicationParams.getScope(),
                 applicationParams.getClientSecret()
             ).getRequest());
     }
+
+    @Override
+    public HearingManagementInterfaceResponse createHearingRequest(JsonNode data) {
+        String authorization = retrieveAuthToken().getAccessToken();
+        return hmiClient.requestHearing("Bearer " + authorization, applicationParams.getSourceSystem(),
+                                        applicationParams.getDestinationSystem(), LocalDateTime.now().format(formatter),
```

Comment:
```suggestion
                                        applicationParams.getDestinationSystem(), Instant.now().toString(),
```
Since it's expecting a UTC format, let's use `Instant.now()`.